docs/orchestrating-elastic-stack-applications/security/managing-authentication-for-multiple-stacks/oidc-stack-config-policy.asciidoc

```diff
@@ -105,8 +105,6 @@ spec:
   resourceSelector:
     matchLabels:
       env: my-label
-  secureSettings:
-  - secretName: oidc-secret
```

Comment:
Do we have a preference for removing this vs noting that this is deprecated, but still "technically valid"?

Reply:
In this case I want to remove it, because we are setting it twice in this example. That is confusing.

Reply:
Ok I see that. 👍